ui/admin/app/templates/scopes/scope.hbs

```diff
@@ -199,6 +199,16 @@
         {{/unless}}
       </Rose::Nav::Sidebar>
 
+      {{#if (and @model.isGlobal (feature-flag 'ssh-session-recording'))}}
```

Comment:
hmm perhaps this could be made part of an admin level canRead ability in scope.js. Now that you mention it i think there should also be a canRead ability check in line 212 and line 222 as well since those nav bar links take users directly to the scope settings pages.

Reply:
actually, nvm i had forgotten that if a scope has no read permissions then the form isn't shown and users see a message. So I don't think that that check fits into any abilities...